src/app_neo4j_queries.py

```diff
@@ -626,8 +579,9 @@ def get_prov_info(neo4j_driver, param_dict, published_only):
     if published_only:
         published_only_query_string = f" AND toUpper(ds.status) = 'PUBLISHED'"
         published_only_revisions_string = f" WHERE toUpper(rev.status) = 'PUBLISHED'"
-    query = (f"MATCH (ds:Dataset)<-[:ACTIVITY_OUTPUT]-(a)<-[:ACTIVITY_INPUT]-(firstSample:Sample)<-[*]-(donor:Donor)"
-             f"WHERE not (ds)-[:REVISION_OF]->(:Dataset)"
+    query = (f"MATCH (ds:Dataset)<-[:ACTIVITY_OUTPUT]-(a:Activity)<-[*]-(firstSample:Sample)<-[*]-(donor:Donor)"
+             f" WHERE not (ds)-[:REVISION_OF]->(:Dataset)"
+             f" AND NOT toLower(a.creation_action) ENDS WITH 'process'"
```

Comment:
The cypher query Bill used in https://github.com/hubmapconsortium/entity-api/issues/534 only finds the derived/processed datasets. However, you are using negation here, so you may need to double check and exclude the “Multi-Assay Split”.

Reply:
Confirmed with Bill:
>“Multi-Assay Split” datasets are primary datasets.